merlin-server/src/test/java/gov/nasa/jpl/aerie/merlin/server/mocks/StubMissionModelService.java

```diff
@@ -28,7 +29,9 @@
   public static final String NONEXISTENT_ACTIVITY_TYPE = "no-activity";
   public static final ActivityType EXISTENT_ACTIVITY = new ActivityType(
       EXISTENT_ACTIVITY_TYPE,
-      List.of(new Parameter("Param", ValueSchema.STRING)), List.of());
+      List.of(new Parameter("Param", ValueSchema.STRING)),
+      List.of(),
+      ValueSchema.UNIT);
```

Comment:
This makes me uncomfortable. We discussed earlier how we are sort of adding a second responsibility onto UNIT with these changes, and here using UNIT as an explicit activity effect model return type feels weird. I don't have a well thought out alternative or solution, but perhaps we need to continue/finish our discussion on UNIT.